packages/libedit/build.sh

```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=http://thrysoee.dk/editline/
+TERMUX_PKG_DESCRIPTION="Command line editor library provides generic line editing, history, and tokenization functions."
+TERMUX_PKG_VERSION=20160903-3.1
+TERMUX_PKG_SRCURL=http://thrysoee.dk/editline/libedit-${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=0ccbd2e7d46097f136fcb1aaa0d5bc24e23bb73f57d25bee5a852a683eaa7567
+TERMUX_PKG_DEPENDS="libandroid-support, ncurses"
+
+termux_step_pre_configure() {
+    CFLAGS+=" -D__STDC_ISO_10646__=201103L -DNBBY=CHAR_BIT"
```

Comment:
1.  __STDC_ISO_10646__ is about unicode support for wchar_t
    libedit will check this or #error, refer from [busybox](http://lists.busybox.net/pipermail/buildroot/2016-January/149100.html) to define a value for it. 
2. NBBY is number of bits per byte definition
    Seems that from BSD, glibc header has this compatible definition, which is not found in NDK.